services/ui-src/src/utils/auth/userProvider.tsx

```diff
@@ -38,7 +38,7 @@ export const UserProvider = ({ children }: Props) => {
       const payload = session.getIdToken().payload;
       const { email, given_name, family_name } = payload;
       // "custom:cms_roles" is an string of concat roles so we need to check for the one applicable to MCR
-      const cms_role = (payload["custom:cms_roles"] as string) ?? "";
+      const cms_role = payload["custom:cms_roles"] as string;
```

Comment:
Is the `??` nullish coalescing here necessary? We're not doing it for any of the other payload items and it should always come back with a role, otherwise IDM wouldn't let them sign in.

Reply:
Reason I added this here is a typeerror when it's not coalesced, if I remember correctly. Maybe there's a better way to handle it. I think it expects `string | undefined` if you don't do this, which the MCRUser in userContext doesn't like

Reply:
What if we just specified `string | undefined` here and let the errors fly as they may?